docs/docs/reference/other-new-features/indentation.md

```diff
@@ -145,14 +136,15 @@ package p:
 package q:
   def b = 2
 ```
+In each case, the `:` at the end of line can be replaced without change of meaning by a pair of braces that enclose the following indented definition(s).
 
 The syntax changes allowing this are as follows:
 
 ```
-TemplateBody ::=  [colonEol] ‘{’ [SelfType] TemplateStat {semi TemplateStat} ‘}’
-EnumBody     ::=  [colonEol] ‘{’ [SelfType] EnumStat {semi EnumStat} ‘}’
-Packaging    ::=  ‘package’ QualId [colonEol] ‘{’ TopStatSeq ‘}’
-RefinedType  ::=  AnnotType {[colonEol] Refinement}
+Template    ::=  InheritClauses [colonEol] [TemplateBody]
+EnumDef     ::=  id ClassConstr InheritClauses [colonEol] EnumBody
+Packaging   ::=  ‘package’ QualId [nl | colonEol] ‘{’ TopStatSeq ‘}’
```

Comment:
Maybe I am missing something, but I cannot recognize the `colonEol` in the parser code for packages:

https://github.com/lampepfl/dotty/blob/44514a34943134f7072881c63502d3f773204f1b/compiler/src/dotty/tools/dotc/parsing/Parsers.scala#L3720-L3724

Is `rewriteWithColon` doing this?

The comment above also does not reflect the colon: 

https://github.com/lampepfl/dotty/blob/44514a34943134f7072881c63502d3f773204f1b/compiler/src/dotty/tools/dotc/parsing/Parsers.scala#L3718

Reply:
It's in `possibleTemplateStart()`. I added a test to verify that `:` is indeed significant.